webclient/src/types/languages.ts

```diff
@@ -1,11 +1,13 @@
 import { CountryLabel } from './countries';
 
 export enum Language {
-  'en-US' = 'en-US',
-  'es-ES' = 'es-ES',
+  'en' = 'en',
```

Comment:
the client currently uses en-US but fr and de for most languages, we also have pt-BR and zh-hans as examples, yet on the client this is all internal, I recommend adding what the language calls itself as a translatable tag

Reply:
region-based translations have been another TBD discussion item, lets make it a follow up.